src/viztracer/modules/vcompressor/vc_dump.c

```diff
@@ -258,3 +264,117 @@ load_events_from_file(FILE* fptr)
     return parsed_events;
 
 }
+
+
+int dump_file_info(PyObject* file_info, FILE* fptr){
+    const char * buffer = NULL;
+    uint64_t content_length = 0;
+    uint64_t compression_length = 0;
+    PyObject* json_ret = NULL;
+    PyObject* zlib_ret = NULL;
+    PyObject* bytes_data = NULL;
+    PyObject* dumps_func = PyObject_GetAttrString(json_module, "dumps");
+    PyObject* json_args = PyTuple_New(1);
+    PyObject* compress_func = PyObject_GetAttrString(zlib_module, "compress");
+    PyObject* zlib_args = PyTuple_New(1);
+
+    if (!dumps_func) {
+        perror("Failed to access json.dumps()");
+        exit(-1);
+    }
+
+    if (!compress_func) {
+        perror("Failed to access zlib.compress()");
+        exit(-1);
+    }
+    
+    // json dumps file_info
+    PyTuple_SetItem(json_args, 0, file_info);
+    Py_INCREF(file_info); 
```

Comment:
Could you add a comment here indicating that `PyTuple_SetItem` steals reference so we need to do `Py_INCREF`?